packages/node_modules/@webex/plugin-rooms/src/rooms.js

```diff
@@ -435,7 +435,7 @@ const Rooms = WebexPlugin.extend({
   getRoomEvent(webex, activity, event) {
     try {
       const sdkEvent = cloneDeep(this.eventEnvelope);
-      const cluster = getHydraClusterString(webex, activity.object.url);
+      const cluster = getHydraClusterString(webex, activity.url);
```

Comment:
Where did this change come from?

Reply:
I'm guessing the convo team wanted to remove "unnecessary" properties. I'm getting a `url` property from a place I know won't go away.